qa/rpc-tests/wallet-hd.py

```diff
@@ -56,20 +60,19 @@ def run_test (self):
             self.nodes[0].generate(1)
         self.nodes[0].sendtoaddress(non_hd_add, 1)
         self.nodes[0].generate(1)
-
         self.sync_all()
-        #assert_equal(self.nodes[1].getbalance(), num_hd_adds + 1)
+        assert_equal(self.nodes[1].getbalance()["bitcoin"], NUM_HD_ADDS + 1)
 
         print("Restore backup ...")
         self.stop_node(1)
         os.remove(self.options.tmpdir + "/node1/elementsregtest/wallet.dat")
         shutil.copyfile(tmpdir + "/hd.bak", tmpdir + "/node1/elementsregtest/wallet.dat")
         self.nodes[1] = start_node(1, self.options.tmpdir, self.node_args[1])
-        #connect_nodes_bi(self.nodes, 0, 1)
+        connect_nodes_bi(self.nodes, 0, 1)
 
         # Assert that derivation is deterministic
         hd_add_2 = None
-        for _ in range(num_hd_adds):
+        for _ in range(NUM_HD_ADDS):
```

Comment:
Can we s/_/i/ like in https://github.com/bitcoin/bitcoin/commit/979f59850c72624137d25f80be4188c3ba6b5fa0 ?